Gemfile.lock

```diff
@@ -3,23 +3,22 @@ GEM
   specs:
```

Comment:
I don't think we should be updating these.

Reply:
Reverted: https://github.com/cerner/fhir.cerner.com/commit/0adab3f661325249f23031c7415eae48d2bc9929